.github/workflows/test-docs.yml

```
@@ -46,6 +46,7 @@ jobs:
         env:
           DJANGO_SETTINGS_MODULE: 'umap.tests.settings'
           UMAP_SETTINGS: 'umap/tests/settings.py'
+          PLAYWRIGHT_TIMEOUT: '10000'
```

I think we can bump it safely there, it's so annoying when there is a failure for that…

```suggestion
          PLAYWRIGHT_TIMEOUT: '20000'
```